### PR TITLE
[501] Add Accessibility, Cookie  & Privacy Policy pages

### PR DESCRIPTION
**Pull request by aliuk2012**
### Changes proposed in this pull request
- Replaces hardcoded footer with GOV.UK Components Footer (there is a bug with the styling of the license footer link but I have submitted a PR to fix it https://github.com/DFE-Digital/govuk-components/pull/69)
- Adds draft accessibility statement
- Adds draft cookie policy page, this will need to be reviewed once we have finished adding dfe signin or start relying on cookies for specific features
- Adds draft privacy policy

### Guidance to review

Visit https://dfe-twd-rttd-pr-180.herokuapp.com/
- Click on Accessibility link in the footer
- See the Accessibility statement page https://dfe-twd-rttd-pr-180.herokuapp.com/accessibility
- Click on Cookies link in the footer
- See the Cookies policy page https://dfe-twd-rttd-pr-180.herokuapp.com/cookies
- Click on Privacy policy link in the footer
- See the Privacy policy page https://dfe-twd-rttd-pr-180.herokuapp.com/privacy-policy

### CODE REVIEW

**Comment by aliuk2012** [comment]
[Implement cookie statement](https://trello.com/c/ncpLlTOL/497-implement-cookie-statement)

**Comment by aliuk2012** [comment]
[Implement accessibility statement](https://trello.com/c/LCXCjwq9/501-implement-accessibility-statement)

**Comment by aliuk2012** [comment]
[Implement privacy policy](https://trello.com/c/9d3SkR7m/511-implement-privacy-policy)

File: app/views/pages/accessibility.html.erb [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<%= render_component PageTitle::View.new(title: "pages.accessibility") %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
```
Comment (edwardhorsford) [inline app/views/pages/accessibility.html.erb:4]:
```suggestion
  <div class="govuk-grid-column-two-thirds-from-desktop">
```

File: app/views/pages/privacy_policy.html.erb [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+      Data Protection Legislation (GDPR) as a data controller for their information.
+    </p>
+
+    <p class="govuk-body govuk-!-margin-bottom-7">
+      You can
+      <%= govuk_link_to "read about your data protection obligations by visiting the Information Commissioner’s Office (ICO) website",
+                        "https://ico.org.uk/for-organisations/" %>.
+    </p>
+
+    <h2 class="govuk-heading-m ">
```
Comment (edwardhorsford) [inline app/views/pages/privacy_policy.html.erb:31]:
```suggestion
    <h2 class="govuk-heading-m">
```

File: app/views/pages/privacy_policy.html.erb [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+      Our lawful basis for processing your personal data
+    </h2>
+
+    <p class="govuk-body">
+      Running and improving the service is a public task, as defined in data protection law. However, we ask
+      for your consent to take part in user research.
+    </p>
+
+    <p class="govuk-body govuk-!-margin-bottom-7">
+      <%= govuk_link_to "Read about consent and public task on the Information Commissioner's Office (ICO) website",
```
Comment (edwardhorsford) [inline app/views/pages/privacy_policy.html.erb:92]:
```suggestion
      <%= govuk_link_to "Read about consent and public task on the Information Commissioner’s Office (ICO) website",
```

File: app/views/pages/privacy_policy.html.erb [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+    </p>
+
+    <h3 class="govuk-heading-s">User research tools</h3>
+
+    <p class="govuk-body">
+      If you agree to take part in user research, we’ll ask if you consent to use Lookback to carry out and
+      record a video session with you.</p>
+
+    <p class="govuk-body govuk-!-margin-bottom-7">
+      <%= govuk_link_to "Visit Lookback's website to read about how they use your data (as a “participant”)", "https://lookback.io/privacy/" %>
```
Comment (edwardhorsford) [inline app/views/pages/privacy_policy.html.erb:156]:
```suggestion
      <%= govuk_link_to "Visit Lookback’s website to read about how they use your data (as a “participant”)", "https://lookback.io/privacy/" %>
```

File: app/webpacker/styles/application.scss [hunk 5]
```diff
@@ -38,3 +38,18 @@ a[href="#"] {
     font-weight: $govuk-font-weight-bold;
   }
 }
+
+
+// Dashed list (used in Contents links list)
+
+.govuk-list--dash li {
```
Comment (edwardhorsford) [inline app/webpacker/styles/application.scss:45]: Should this be `app-list--dash` ?

Comment (aliuk2012) [reply app/webpacker/styles/application.scss:45]: 👍 need to update the prototype its also uses this class name

Comment (aliuk2012) [inline app/webpacker/styles/application.scss:45]:
```suggestion
.app-list--dash li {
```

File: app/views/pages/privacy_policy.html.erb [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+      You can
+      <%= govuk_link_to "read about your data protection obligations by visiting the Information Commissioner’s Office (ICO) website",
+                        "https://ico.org.uk/for-organisations/" %>.
+    </p>
+
+    <h2 class="govuk-heading-m">
+      In this privacy policy
+    </h2>
+
+    <ul class="govuk-list govuk-list--dash govuk-!-margin-bottom-8">
```
Comment (aliuk2012) [inline app/views/pages/privacy_policy.html.erb:35]:
```suggestion
    <ul class="govuk-list app-list--dash govuk-!-margin-bottom-8">
```

**Comment by defong** [comment]
> [Code Smell](https://sonarcloud.io/project/issues?id=DFE-Digital_register-trainee-teacher-data&pullRequest=180&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=DFE-Digital_register-trainee-teacher-data&pullRequest=180&resolved=false&types=CODE_SMELL) [2 Code Smells](https://sonarcloud.io/project/issues?id=DFE-Digital_register-trainee-teacher-data&pullRequest=180&resolved=false&types=CODE_SMELL)
> 

Is there a way to configure this?

**Comment by EmmaFrith** [comment]
there's some funny link text in the privacy policy: Contact the Register trainee teachers team at registerateacher@digital.education.gov.uk?subject=My%20personal%20data%20 if you have any concerns about your personal data

**Comment by EmmaFrith** [comment]
@aliuk2012 could you remove this sentence for now? and we can update when we know how accessible the service is: 

This website is partially compliant with the Web Content Accessibility Guidelines (WCAG) version 2.1 AA standard, due to the non-compliances listed below.

**Review by EmmaFrith — APPROVED** [review]

File: app/views/pages/accessibility.html.erb [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+      Regulations 2018.
+    </p>
+
+    <p class="govuk-body">
+      This website is partially compliant with the <%= govuk_link_to "Web Content Accessibility Guidelines (WCAG) version 2.1", "https://www.w3.org/TR/WCAG21/" %>
+      AA standard, due to the non-compliances listed below.
+    </p>
+
+    <p class="govuk-body">
+      The service was designed using the <%= govuk_link_to "GOV.UK design system", "https://design-system.service.gov.uk/accessibility/" %>,
```
Comment (edwardhorsford) [inline app/views/pages/accessibility.html.erb:43]:
```suggestion
      The service was designed using the <%= govuk_link_to "GOV.UK Design System", "https://design-system.service.gov.uk/accessibility/" %>,
```

**Review by edwardhorsford — APPROVED** [review]

File: app/views/pages/privacy_policy.html.erb [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+      You have rights under the Data Protection Act 2018, such as the right to ask for your data to be erased.
+    </p>
+
+    <p class="govuk-body">
+      <%= govuk_link_to "Find out about your rights by visiting the Information Commissioner’s Office (ICO) website", "https://ico.org.uk/" %>
+    </p>
+
+    <p class="govuk-body">
+      Contact the <%= t('service_name') %> team at
+      <%= govuk_mail_to "registerateacher@digital.education.gov.uk" , "registerateacher@digital.education.gov.uk", subject: "My personal data" %>
```
Comment (aliuk2012) [inline app/views/pages/privacy_policy.html.erb:184]: @EmmaFrith thanks for pointing out the long encrypted test you were seeing on the page. I've fixed this now but it was basically presetting the subject of the email if the user clicked on the link.

**Pull request merged**